.github/pull_request_template.md

```diff
@@ -1,4 +1,6 @@
-<sub>By submitting this pull request you agree that all contributions to this project are made under the MIT license.</sub>
+## Summary
+
+<!-- Describe these changes -->
```

Comment:
The main things I like to see in a PR are a description of the problems and how the changes address them. Without some direction, people tend to jump straight to what the changes are and fail to justify why they are necessary. That's why I put the "Issues" section first.

Perhaps the comment under the "Issues" section does not adequately emphasize that.